code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm

```diff
@@ -775,7 +775,7 @@
 
 /datum/reagent/drink/dry_ramen
 	name = "Dry Ramen"
-	description = "Space age food, since August 25, 1958. Contains dried noodles, vegetables, and chemicals that boil in contact with water."
+	description = "Ramen that never spoils! since August 25, 1910. Contains dried noodles, vegetables, and chemicals that boil in contact with water."
```

Comment:
```suggestion
	description = "Ramen that never spoils! since August 25, 1958. Contains dried noodles, vegetables, and chemicals that boil in contact with water."
```

Reply:
Dry Ramen was discovered in 1910.